src/appBootstrapper.jsx

```diff
@@ -212,4 +194,21 @@ if (__DEBUG__ && module.hot) {
     module.hot.accept('./routes', () => render());
 }
 
-$(document).ready(() => render());
+$(document).ready(async () => {
+
+    // we use rawServerConfig (written by JSP) if it is present
+    // or fetch from config service if not
+    // need to use jsonp, so use jquery
+    let config = window.rawServerConfig || await $.ajax({
```

Comment:
Since we still have jsp (and need it for authorization) we might as well write config directly to it rather than fetching config async unnecessarily.   this is trivial to achieve in jsp